### DISCUSSION TITLE
How to exclude set of words in ignore words

### DISCUSSION THREAD

**Discussion by papampi** [pymedusa/Medusa#10356] [Q&A]
My Medusa keeps snatching "The.Fresh.Prince.of.Bel-Air" episodes for the new "Bel-Air" show, I tried to add "Fresh Prince" and also "The Fresh Prince of" to the ignore list of this show but its not working.
I'm afraid if I set single words "Fresh" and "Prince" it may ignore upcoming episodes if they have those in episode name.

``` 
2022-02-17 14:35:51 INFO	FORCEDSEARCHQUEUE-MANUAL-388151 :: jacket-pb :: [c797b83] Picked The.Fresh.Prince.of.Bel-Air.S01E04.1080p.HEVC.x265-MeGusta as the best result.
```

Anyway to correct this issue?

**Comment by p0psicles** [comment]
What does the release name in the guessit test tool return.

Tip: there is a guessit tab in config - postprocess

**Reply by papampi** [reply]
{
    "title": "Bel-Air",
    "type": "episode",
    "parsing_time": 0.06328701972961426
}

And with episode:
{
    "title": "Bel-Air",
    "season": 1,
    "episode": 4,
    "type": "episode",
    "parsing_time": 0.08098053932189941
}

**Reply by papampi** [reply]
Guessit resolve "The Fresh Prince of Bel-Air" to "Bel-Air" too
The.Fresh.Prince.of.Bel-Air.S01E04.1080p.HEVC.x265-MeGusta.mkv
{
    "title": "Bel-Air",
    "season": 1,
    "episode": 4,
    "screen_size": "1080p",
    "video_codec": "H.265",
    "video_profile": "High Efficiency Video Coding",
    "video_encoder": "x265",
    "release_group": "MeGusta",
    "container": "mkv",
    "type": "episode",
    "parsing_time": 0.12578225135803223
}

**Reply by p0psicles** [reply]
When I use the latest develop branch. I get this:
for `The.Fresh.Prince.of.Bel-Air.S01E04.1080p.HEVC.x265-MeGusta.mkv`
```
{
    "title": "The Fresh Prince of Bel-Air",
    "season": 1,
    "episode": 4,
    "screen_size": "1080p",
    "video_codec": "H.265",
    "video_profile": "High Efficiency Video Coding",
    "video_encoder": "x265",
    "release_group": "MeGusta",
    "container": "mkv",
    "mimetype": "video/x-matroska",
    "type": "episode",
    "parsing_time": 0.0025835037231445312
}
```

**Reply by p0psicles** [reply]
Never mind. I see what's going on. When you add the show `bel-air`, that title is added to the expected titles. And it's matching towards that show. It's added to expected shows because there is a `-` in it.
